.changeset/neat-planes-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-app-api': patch
+---
+
+fix: loading lazy messages when init app and simplify i18n options types
```

Comment:
Let's skip changesets for the followup changes for now, until i18n has been released in main line

Reply:
1